README.md

```diff
@@ -44,7 +44,7 @@ The result looks like this:
 ```json
 {
   "title": "Thunder (mascot)",
-  "content": "<div><div><p>This is the content of the page!</div></div>",
+  "content": "... <p><b>Thunder</b> is the <a href="https://en.wikipedia.org/wiki/Stage_name">stage name</a> for th...e",
```

Comment:
Small typo... `for th...e` should be `for the...`

Reply:
Also might have to escape the double quotes around the `href` attr.

Reply:
👍